code/game/machinery/doors/firedoor.dm

```diff
@@ -61,17 +62,16 @@
 	. = ..()
 	COOLDOWN_START(src, detect_cooldown, DETECT_COOLDOWN_STEP_TIME)
 	soundloop = new(src, FALSE)
-	AddElement(/datum/element/atmos_sensitive, mapload)
 	CalculateAffectingAreas()
-	my_area = get_area(src)
+	AddElement(/datum/element/atmos_sensitive, mapload)
```

Comment:
This isn't strictly a you thing, but we don't actually implement any of the atmos sensitive procs on this door. I think I miscommunicated something to zxaber, but I don't think this is required